CLAUDE.md

```
@@ -229,10 +235,11 @@ const response = await fetch("/api/files/${fileId}", { method: "DELETE" });
 - **ブランチ命名**: `feat/*`, `fix/*`, `chore/*`, `refactor/*`
 - **push前 Done条件（品質ゲート）**:
   ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)